.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2.1
+orbs:
+  ruby: circleci/ruby@1.8.0
+
+jobs:
+  build:
+    docker:
+      - image: "cimg/openjdk:17.0.3"
+    steps:
+      - checkout
+      - run: gradle build
+  unitTests:
+    docker:
+      - image: "cimg/openjdk:17.0.3"
+    steps:
+      - checkout
+      - run: gradle test
+  acceptanceTests:
+    docker:
+      - image: "cimg/openjdk:17.0.3"
+    steps:
+      - checkout
+      - run:
+          command: gradle run
+          background: true
+      - run: git submodule update --init --recursive
+      - ruby/install:
+          version: '2.6.8'
+      - run:
+          command: |
+            cd http_server_spec
+            gem install bundler
+            bundle install --path vendor/bundle
+            sudo apt install lsof
```

Comment:
Not suggesting a change. I'm curious why `lsof` is needed.

Reply:
Ignore my last comment. You mentioned it before and I forgot. For the other mentors:
<img width="853" alt="Screen Shot 2022-06-14 at 5 36 01 PM" src="https://user-images.githubusercontent.com/9084699/173693393-7746408f-e14e-4a22-bc3a-d9a75fd2f4d1.png">